THIRD-PARTY.md

```diff
@@ -307,4 +307,3 @@ Public domain
 -------------------------------------------------------------------------
 
 If we have left anything out, it is unintentional. Please let us know.
-
```

Comment:
I kinda wish PRs would not make unrelated formatting changes to other parts of the codebase, but whatever, this is harmless.